apps/api/src/helper/object.helper.ts

```diff
@@ -1,35 +1,5 @@
 import { Big } from 'big.js';
-import { cloneDeep, isArray, isObject } from 'lodash';
-
-export function hasNotDefinedValuesInObject(aObject: Object): boolean {
-  for (const key in aObject) {
-    if (aObject[key] === null || aObject[key] === undefined) {
-      return true;
-    } else if (isObject(aObject[key])) {
-      return hasNotDefinedValuesInObject(aObject[key]);
-    }
-  }
-
-  return false;
-}
-
-export function nullifyValuesInObject<T>(aObject: T, keys: string[]): T {
-  const object = cloneDeep(aObject);
-
-  if (object) {
-    keys.forEach((key) => {
-      object[key] = null;
-    });
-  }
-
-  return object;
-}
-
-export function nullifyValuesInObjects<T>(aObjects: T[], keys: string[]): T[] {
-  return aObjects.map((object) => {
-    return nullifyValuesInObject(object, keys);
-  });
-}
+import { isArray, isObject } from 'lodash';
 
 export function redactAttributes({
```

Comment:
Can you maybe keep the original function and let them run against each other in the test (to compare performances)?